bugbug/test_scheduling_features.py

```diff
@@ -108,3 +108,17 @@ def __call__(self, test_job, **kwargs):
                 "failures_past_56_pushes_in_components"
             ],
         }
+
+class arch(object):
+    def __call__(self, test_job, **kwargs):
+        archs = []
+        architectures = (("arm",), ("64", "aarch64"), ("32", "x86", "i386"))
+        for arcs in architectures:
+            for a in arcs:
+                if a in test_job["name"][: test_job["name"].index("/")]:
+                    archs.append(arcs[0])
+                    break
```

Comment:
You are kind of "cheating" here, this way the assertion I asked you to add https://github.com/mozilla/bugbug/pull/1040/files#r339354740 will clearly be met, but simply because you are not trying other matches.

Reply:
I used the implementation of [`platform`](https://github.com/mozilla/bugbug/blob/2740992b9159747c6168dee56116789bb134a151/bugbug/test_scheduling_features.py#L12) as a reference. 

So, if a `test_job`has multiple architectures in its name, you want this function to throw an assertion error?